packages/cli/lib/connectors.json

```diff
@@ -820,22 +820,6 @@
     },
     "supportedByStrongLoop": false
   },
-  "mail": {
```

Comment:
I couldn't find this package so I removed it. Let me know if this should be reverted.